plugins/RocketsPlugin/jsonSerialization.h

```diff
@@ -126,6 +137,25 @@ STATICJSON_DECLARE_ENUM(brayns::TextureType,
 
 namespace staticjson
 {
+inline void init(brayns::PropertyMap* /*g*/, ObjectHandler* h)
+{
+    // beware that this should only be used for creating JSON RPC schema which
+    // uses PropertyMap as an argument and a oneOf list is not possible. To
+    // properly serialize a property map to/from JSON, use from_json/to_json
+    // from jsonPropertyMap.h directly.
+    h->set_flags(Flags::DisallowUnknownKey);
+}
+inline void init(brayns::ModelProperties* s, ObjectHandler* h)
+{
+    h->add_property("id", &s->modelID);
+    h->add_property("properties", &s->properties);
```

Comment:
Are the params for set model props:
```js
{
  id: 1,
  properties: {
    ping: true
  }
}
```
or:
```js
{
  id: 1,
  ping: true
}
```
?

Second would make more sense, unless that's difficult to implement.

Reply:
It's first option for now. Ideally, the RPC would have 2 arguments, but that's not supported right now in Rockets iirc. The second option is quite unnatural, as the 'id' is not a property of the model properties, it's only needed to find the model.

Reply:
Ok. Does this apply for broadcast and get or is it only the set?

Reply:
set- and broadcast. get- is just the properties, the ID is the input for the get- request.